engine/src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -40,6 +40,7 @@ case class WorkflowDescriptor(id: WorkflowId,
                               backend: Backend,
                               configCallCaching: Boolean,
                               lookupDockerHash: Boolean,
+                              configWorkflowFailureMode: Boolean,
```

Comment:
OTOH, agile is all about incremental changes and not over-engineering solutions for a "maybe, possibly..." future scenario :)